### PR TITLE
replace deprecated class from commons-lang

### PR DESCRIPTION
**Pull request by elharo**

### CODE REVIEW

**Review by michael-o — APPROVED** [review]

**Review by slachiewicz — APPROVED** [review]

**Comment by elharo** [comment]
Build windows-jdk11 looks borked again :-(

File: doxia-core/pom.xml [hunk 1]
```diff
@@ -59,6 +59,11 @@ under the License.
       <artifactId>commons-lang3</artifactId>
       <version>3.8.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-text</artifactId>
+      <version>1.3</version>
```
Comment (elilja) [inline doxia-core/pom.xml:65]: Why such an old version of commons-text?

Comment (elharo) [reply doxia-core/pom.xml:65]: Because everything newer requires Java 8. :-(

Comment (elilja) [reply doxia-core/pom.xml:65]: That's fine since some time ago, I believe. You may require Java 8 in plugins and components etc for the Maven eco system

**Pull request merged**